views/funcoes_request_response.py

```diff
@@ -216,3 +216,25 @@ def editar_global_settings(request):
         info_editar_settings["prazo_nota"],
     )
     return JsonResponse({"mensagem": "concluido"})
+
+def listar_fee_alunos_turma(request, id_turma):
+    alunos = gerenciador_turmas_alunos.listar_alunos_turma(id_turma)
+    for id_aluno in alunos.keys():
+        fee = gerenciador_notas._calcular_fee_turma_aluno(id_aluno=id_aluno,id_turma=id_turma)
+        alunos[id_aluno]["fee"] = fee
+    return JsonResponse(alunos)
+
+def listar_fee_turmas(request):
+    turmas = gerenciador_turmas.busca_turmas()
```

Comment:
Isso é regra de negócio, precisa ir para dentro de um módulo. A view não deve fazer lógicas longas, apenas receber um request, tratar o dado para enviar para o service, onde é aplicado regra de negócio, e depois para o operador do banco. então nesse arquivo aqui regra_de_negocio/service.py você cria um listar_fee_turmas_svc. Coloca essa parte da lógica lá. Como não tem nenhuma operação de salvar no banco, nem editar, ele pode ficar lá. 
` for id_turma in turmas.keys():
        alunos = gerenciador_turmas_alunos.listar_alunos_turma(id_turma)
        fee_turma = []
        for id_aluno in alunos.keys():
            fee = gerenciador_notas._calcular_fee_turma_aluno(id_aluno=id_aluno,id_turma=id_turma)
            fee_turma.append(fee)
        if len(fee_turma) == 0:
            turmas[id_turma]['fee'] = 0.0   
        else:
            media_fee = sum(fee_turma) / len(fee_turma)
            turmas[id_turma]['fee'] = round(media_fee, 2)`